testing/integration/install_test.go

```diff
@@ -53,8 +53,8 @@ func TestInstallWithoutBasePath(t *testing.T) {
 	}
 
 	topPath := filepath.Join(defaultBasePath, "Elastic", "Agent")
-	_, err = os.Stat(topPath)
-	require.True(t, os.IsNotExist(err))
+	err = os.RemoveAll(topPath)
+	require.NoError(t, err, "failed to remove %q. The test requires this path not to exist.")
```

Comment:
Doesn't this change the tests? The test is checking that the uninstall did the correct thing. This basically ignores if it does and just removes itself. That seems to be masking a possible error.

Reply:
No, it's a pre-condition for the test as it's testing the agent successfully installs without the base path (`TestInstallWithoutBasePath`).
Being it a precondition, it'd not be fulfilled only if another test would let the base path there, therefore I don't see why the test preparation couldn't ensure it's absent.